test/unit/spec/v1/EnrollSms_spec.js

```diff
@@ -123,24 +123,21 @@ Expect.describe('EnrollSms', function() {
 
   function expectResendButton(test) {
     const button = test.form.sendCodeButton();
-
-    expect(button.trimmedText()).toEqual('Re-send code');
-    expect(button.hasClass('button-primary')).toBe(false);
+    expect(button.innerHTML).toEqual('Re-send code');
```

Comment:
nit: use `innerText` if possible on these instead of `innerHTML`

Reply:
`innerText` does indeed fail, it looks to be due to the use of jsdom, this would explain why this file uses `innerHTML` instead. 

See: https://github.com/jsdom/jsdom/issues/1245